### PR TITLE
2063 feat button icon

### LINKED ISSUE #2063: [Feature?]: button icon followup
### Duplicates

- [X] I have searched the existing issues

### Latest version

- [X] I have tested the latest version

### Summary 💡

#2044 added the button component ref to the v1 docs. An emoji vs an icon was included in the button with icon example - it should be swapped with the `Placeholder` icon for a11y reasons

### Examples 🌈

_No response_

### Motivation 🔦

a11y

### PR DESCRIPTION
**Pull request by kathydlavoie-ps**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

closes #2063

## What is the new behavior?

Replaces emojis as start and end icons in button docs with placeholder icon for a11y purposes

## Other information
Screenshot 2024-02-05 at 4 36 10 PM

### CODE REVIEW

File: docs/package.json [hunk 1]
```diff
@@ -23,7 +23,8 @@
     "sharp": "^0.32.5",
     "nstall": "^0.2.0",
     "react-live": "^4.1.5",
-    "@pluralsight/react": "workspace:*"
+    "@pluralsight/react": "workspace:*",
+    "@pluralsight/icons": "workspace:*"
```
Comment (caseybaggz) [inline docs/package.json:27]: Can we test removing icons? You shouldn't need it in the latest PR we merged in yesterday.

File: docs/src/components/LiveCodeExamples/Button/LiveButton.tsx [hunk 2]
```diff
@@ -2,6 +2,7 @@ import { Button } from '@pluralsight/react'
 import { button } from '@/styled-system/recipes'
 import { css, cx } from '@/styled-system/css'
 import LiveCodeEditor from '../LiveCodeEditor'
+import { PlaceholderIcon } from '@pluralsight/icons'
```
Comment (caseybaggz) [inline docs/src/components/LiveCodeExamples/Button/LiveButton.tsx:5]:
```suggestion
import { PlaceholderIcon } from '@pluralsight/react/icons'
```
This should work but is being funky. Just want to test.

Comment (kathydlavoie-ps) [reply docs/src/components/LiveCodeExamples/Button/LiveButton.tsx]: sure thing!

**Review by caseybaggz — APPROVED** [review]
🚢

**Pull request merged**